test/bin/scenario.sh

```diff
@@ -88,10 +88,18 @@ prepare_kickstart() {
 # Show the IP address of the VM
 function get_vm_ip {
     local vmname="${1}"
-    sudo virsh domifaddr "${vmname}" \
-        | grep vnet \
-        | awk '{print $4}' \
-        | cut -f1 -d/
+    start=$(date +%s)
```

Comment:
`vmname` and `start` can be `local -r`, right?

Reply:
Done.